shelldio.sh

```diff
@@ -59,9 +80,11 @@ option_detail() {
 	
 	-h, --help: Εμφανίζει πληροφορίες για την χρήση της εφαρμογής
 	
-	-l, --list: Εμφανίζει την λίστα με τους ραδιοφωνικούς σταθμούς.
+	-l, --list: Εμφανίζει την λίστα με τους ραδιοφωνικούς σταθμούς
 	
 	-r, --remove: Διαγράφει σταθμούς της επιλογής σας από το my_stations.txt
+
+	--reset: Καθαρίζει τη λίστα αγαπημένων, διαγράφοντας το αρχείο my_stations.txt
```

Comment:
Στην περίπτωση που αλλάξει για κάποιο λόγο το όνομα του αρχείου στο μέλλον, ίσως ξεχάσουν να το αλλάξουν και σε αυτό το σημείο. Χρησιμοποιώντας την μεταβλητή έχεις το κεφάλι σου ήσυχο ;)

```suggestion
	--reset: Καθαρίζει τη λίστα αγαπημένων, διαγράφοντας το αρχείο $my_stations
```

Reply:
Κράτησα το ίδιο με το `-r`. Να τα αλλάξω όλα στη μεταβλητή `$my_stations`;
Το ίδιο και στην add `-a, --add : Δημιουργεί το αρχείο ~/.shelldio/my_stations.txt` ?

Reply:
Άνοιξα ένα issue. είναι καλή ιδέα να γίνει σε όλα

Reply:
@cerebrux based on the thumbs up να υποθέσω ότι θέλεις να το αλλάξω πάντου στο help να χρησιμοποιεί τη μεταβλητή `$my_stations`;

Reply:
Yeap, εξού και άνοιξα issue για να κλείσει αυτόματα με το merge

Reply:
Done!